### ISSUE TITLE
Support package.cfg in rfpkg

### ISSUE THREAD

**Issue by kwizart** [rpmfusion-infra/rfpkg#18]
Recently fedpkg gained support for package.cfg in dist-git
This file allows to submit a build for different koji targets

I want to re-use this feature for the rpi namespace (at least) where I want to have the package built twice (or more)

**Comment by sergiomb2** [comment]
this feature (support for package.cfg in dist-git) could be used for multilibs [1] ?

[1]
https://bugzilla.rpmfusion.org/show_bug.cgi?id=5070

**Comment by kwizart** [comment]
No, that's a separate issue. But on second tough the implementation can share some bits.

With package.cfg in dist-git, the aim is to submit the build twice one for the default target and another for another (twin) target.
So if one is building for epel8, another build will be done for epel8-playground.
This is how it's implemented in epel.

The way I plan to use this feature is with rpi and cuda namespaces at this step.
Basically, I want to submit the build twice for some packages, one using the normal default target and another time using the rpi target.

In this feature, I wish I could use a package.cfg that would have some regexp to determine the appropriate target. (not to hardcode every fedora branches).

The multilibs target is the same principle, but without building the default target.
For example xorg-x11-drv-nvidia is build using the el7-nonfree-multilibs target instead of el7-nonfree default target. So it might be possible to re-use the package.cfg information in the git module rather than building a static list of packages...

**Comment by kwizart** [comment]
RFE forwarded to https://pagure.io/fedpkg/issue/354

**Comment by sergiomb2** [comment]
https://pagure.io/fedpkg/blob/master/f/fedpkg/cli.py#_612     def register_build(self): 

https://pagure.io/fedpkg/blob/master/f/fedpkg/cli.py#_1247     def _build(self, sets=None):

I think is this that is request in this issue, isn't it ?

**Comment by kwizart** [comment]
It's less important since package.cfg in epel8 is deprecated/unused and might disappear at some point...
We might come with our own implementation of how to determine which build needs submitted with an additional target.

**Comment by sergiomb2** [comment]
OK , I'm preparing a new rfpkg version here https://github.com/rpmfusion-infra/rfpkg/commits/fedpkg_sync_2021 
any feedback is welcome

**Comment by kwizart** [comment]
package.cfg isn't current in modern fedpkg/rpkg
